_data/instagram.yml

```diff
@@ -4,6 +4,6 @@ blog:
   link: /blog/methods-and-tools/pdf-generation-with-java/
 
 job:
-  image: /assets/img/pages/headers/public/abstract-art-blur-bright.jpg
-  text: We're hiring a<br /><strong>JavaScript Engineer (m/f/d)</strong>
-  link: https://epages-gmbh.jobs.personio.de/job/201909?language=en
+  image: /assets/img/pages/headers/public/first-hackathon.jpg
+  text: We're hiring a<br /><strong>Webdesigner (m/f/d)</strong>
```

Comment:
I just checked and it seems to be "Web Designer" on most English job pages. I think we should go with that? :)